hapi/hedera-protobufs/services/state/roster/ledger_id.proto

```diff
@@ -0,0 +1,106 @@
+/**
+ * # Ledger ID
+ *
+ * ### Keywords
+ * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+ * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+ * document are to be interpreted as described in
+ * [RFC2119](https://www.ietf.org/rfc/rfc2119) and clarified in
+ * [RFC8174](https://www.ietf.org/rfc/rfc8174).
+ */
+syntax = "proto3";
+
+package com.hedera.hapi.node.state.roster;
+
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.roster">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * A public key that identifies the ledger and can be used to verify ledger signatures.
+ */
+message LedgerId {
+
+  /**
+   * A public key.
+   * This key both identifies the ledger and can be used to verify ledger signatures.
+   * <p>
+   * This value MUST be set.
+   * This value MUST NOT be empty.
+   * This value MUST contain a valid public key.
```

Comment:
This lacks line breaks; without them the produced documents will be concatenated to a single line due to markdown tables.
```suggestion
   * A public key.<br/>
   * This key both identifies the ledger and can be used to verify ledger signatures.
   * <p>
   * This value MUST be set.<br/>
   * This value MUST NOT be empty.<br/>
   * This value MUST contain a valid public key.
```